pkl-core/src/main/java/org/pkl/core/project/Project.java

```diff
@@ -506,6 +509,7 @@ public EvaluatorSettings(
         @Nullable Map<String, String> env,
         @Nullable List<Pattern> allowedModules,
         @Nullable List<Pattern> allowedResources,
+        @Nullable Boolean color,
```

Comment:
Let's remove this line; this class is deprecated and should be considered as not receiving updates.